### PR TITLE
 media-sound/guitarix ebuild updates

### PR DESCRIPTION
**Pull request by puleglot**

### CODE REVIEW

**Comment by puleglot** [comment]
Depends on https://github.com/gentoo/gentoo/pull/12210 (=media-libs/zita-convolver-4*)

File: media-sound/guitarix/guitarix-0.38.1-r1.ebuild [hunk 1]
```diff
@@ -24,7 +24,11 @@ COMMON_DEPEND="dev-cpp/eigen:3
 	dev-cpp/gtkmm:2.4
 	dev-libs/glib:2
 	>=media-libs/libsndfile-1.0.17
-	>=media-libs/zita-convolver-3
+	media-libs/zita-convolver:=
+	|| (
```
Comment (a17r) [inline media-sound/guitarix/guitarix-0.38.1-r1.ebuild]: what makes this block necessary?

Comment (puleglot) [reply media-sound/guitarix/guitarix-0.38.1-r1.ebuild]: This is to apply both slot-operator and version restrictions to the dependency. Build system explicitly supports only zita-convolver-3 and -4. Or should it just depend on `>=media-libs/zita-convolver-3:=` until newer version of zita-convolver is released and added to the repo?

Comment (mgorny) [reply media-sound/guitarix/guitarix-0.38.1-r1.ebuild]: Yes, it should. If new subslot is added, rebuild will be forced and you'll get a nice bugreport ;-P.

**Pull request closed without merging**